### DISCUSSION TITLE
Welcome to Prism Discussions!

### DISCUSSION THREAD

**Discussion by brianlagunas** [PrismLibrary/Prism#2212] [General]
## 👋 Welcome!
  We’re using Discussions as a place for our community to support each other. We hope that you:
  * Ask questions.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

This is meant to be a place for the community to support each other, and not a dedicated support channel. There are no expectations or guarantees of Brian or Dan responding to your questions.

However, if you are a GitHub sponsor, Brian and/or Dan will respond when available.

You can sponsor Brian and Dan here:
- [Sponsor Brian](https://github.com/sponsors/brianlagunas)
- [Sponsor Dan](https://github.com/sponsors/dansiegel)

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AlexHedley** [comment]
So great to have everything in that same place

**Comment by gh33da** [comment]
https://prismlibrary.com/docs/wpf/region-navigation/basic-region-navigation.html <-- Looks like all of the region navigation pages are blank.

**Comment by dansiegel** [comment]
@gh33da the docs have their own [repo](/PrismLibrary/Prism-Documentation) where you can file issues. Also the WPF docs are in the early stages up an update. The docs for Region Navigation are still all there and still relevant under the Legacy section. Those docs were written by the Microsoft team and we're pretty sure they were paid by the word... Feel free to open a PR in the docs repo if you'd like to help translate the verbose legacy docs to a more reader's digest (easy for a newb) version.
